vmdk_plugin/plugin.go

```diff
@@ -281,7 +321,7 @@ func (d *vmdkDriver) Mount(r volume.MountRequest) volume.Response {
 		isReadOnly = true
 	}
 
-	mountpoint, err := d.mountVolume(r.Name, isReadOnly)
+	mountpoint, err := d.mountVolume(r.Name, status["fstype"].(string), isReadOnly)
```

Comment:
syscall.Mount do not support "auto" (which is the mount command doing its own magic). In this case we would have to call the mount command externally.

I like the idea of tracking the fstype in metadata because it enforce strictness, or maybe i did not get the error-prone scenario right, can you provide an example?